src/libraries/System.IO.Hashing/src/System/IO/Hashing/Crc32.cs

```diff
@@ -175,17 +175,17 @@ private static uint Update(uint crc, ReadOnlySpan<byte> source)
 #if NET7_0_OR_GREATER
             if (System.Runtime.Intrinsics.Vector128.IsHardwareAccelerated
                 && System.Runtime.Intrinsics.X86.Sse2.IsSupported
-                && System.Runtime.Intrinsics.X86.Pclmulqdq.IsSupported)
+                && System.Runtime.Intrinsics.X86.Pclmulqdq.IsSupported
```

Comment:
You only need `Pclmulqdq.IsSupported` as it implies `Sse2` support and therefore also implies `Vector128.IsHardwareAccelerated`

Reply:
Done